### PR TITLE
Fixes Ballot tabs on create/delete (fixes #3058)

### LINKED ISSUE #3058: Creating a new election round does no longer jump to it
This is new in beta 4 preview:
- create a first voting round for an election
- afterwards create a second election round

result: the second round is created but its tab is not activated, the tab of the first round is still active.

This is a minor issue but maybe easy to fix.

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
Also some rework in the DetailCtrl. Always use `$scope.assignment` and `assignmentId`.

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Tested. Sometime (after deleting some created tabs) I get a state where no tab is active. Can you catch this (empty) state?

**Comment by FinnStutzenstein** [comment]
Hmm, I cannot recreate this. Please add `$scope.activeTab=0;` in line 555 (in `deleteBallot` before the flag is set).

**Comment by emanuelschuetze** [comment]
Works. As discussed please add `$scope.activetTab = $scope.activetTab - 1;`

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze Done and rebased.

**Comment by FinnStutzenstein** [comment]
Info: You can just test this, if you use the server code from PR #3111 .

File: openslides/assignments/static/js/assignments/site.js [hunk 1]
```diff
@@ -549,7 +552,8 @@ angular.module('OpenSlidesApp.assignments.site', [
         $scope.deleteBallot = function (poll) {
             poll.DSDestroy().then(
                 function (success) {
-                    $scope.activeTab = assignment.polls.length - 1;
+                    scope.activeTab = $scope.activeTab - 1;
```
Comment (emanuelschuetze) [inline openslides/assignments/static/js/assignments/site.js]: typo: $scope

Comment (FinnStutzenstein) [inline openslides/assignments/static/js/assignments/site.js]: Done.

**Pull request merged**